diff --git a/node_modules/@artilleryio/int-core/lib/runner.js b/node_modules/@artilleryio/int-core/lib/runner.js
index 9569231..2d94a44 100644
--- a/node_modules/@artilleryio/int-core/lib/runner.js
+++ b/node_modules/@artilleryio/int-core/lib/runner.js
@@ -54,7 +54,8 @@ function loadEngines(
         if (typeof Engines[engineName] !== 'undefined') {
           Engine = Engines[engineName];
         } else {
-          Engine = require(moduleName);
+          // Engine = require(moduleName);
+          Engine = require(path.join(process.env.ARTILLERY_ENGINE_PATH, moduleName));
         }
         const engine = new Engine(script, ee, engineUtil);
         engine.__name = engineName;
